lib-compose/src/main/java/com/what3words/components/compose/maps/state/camera/W3WGoogleCameraState.kt

```diff
@@ -49,28 +50,33 @@ class W3WGoogleCameraState(override val cameraState: CameraPositionState) :
         )
     }
 
-    override fun getZoomLevel(): Float {
-        return cameraState.position.zoom
-    }
+    @UiThread
```

Comment:
The @UiThread annotation in Android is a contract or indicator, not a guarantee and it does not enforce it at runtime or switch to the main thread automatically. To ensure thread safety and compliance with coroutine rules, please modify the function to explicitly run on the Dispatchers.Main dispatcher